src/components/keypad/pre-algebra-page.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+import React from "react";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+
+import Keys from "../../data/key-configs";
+import {
+    KeypadPageContainer,
+    KeypadButton,
+    SecondaryKeypadButton,
+    KeypadActionButton,
+    PlaceHolderButton,
+} from "./keypad-page-items";
+
+import type {KeyConfig} from "../../data/key-configs";
+
+type Props = {|
+    onClickKey: (keyConfig: string) => void,
```

Comment:
I don't see the import `KeyConfig` type being used anywhere.  Should this be `(keyConfig: KeyConfig) => mixed`?  The `mixed` part isn't necessary here but it's a common pattern we use with React callbacks to allow for easy one-liners where the body might return a value.